physcraper/__init__.py

```diff
@@ -2264,7 +2130,10 @@ def seq_dict_build(self, seq, label, seq_dict):
         :param seq_dict: the tmp_dict generated in add_otu()
         :return: updated seq_dict
         """
-
+        if label == None: #in case of add_otu failure, doean't edit dict 
```

Comment:
This will never happen, the otu_id is assigned in add_otu(), a function that is run before this one. If the function is not run, there is no label to input into this function.

Reply:
I added a line to add_otu to return None if it is not possible to find the ncbi_id, so that we were not passing any otus that don't have an ncbi_id to the filter blast class. But I think deciding how we want to handle otu's without ncbi_ids is connected to the discussion on ln 989.

Reply:
Oh wait - actually get_tax_info_from_acc will throw an error rather than returning None if it can't find a taxon id, so this will never happen, the way the code is currently.